### ISSUE TITLE
Custom CSS Stylesheets

### ISSUE THREAD

**Issue by HarleyJones** [bluesky-social/social-app#5865]
### Describe the Feature

I would like the ability for power users to be able to theme the bluesky app with CSS stylesheets

**Comment by ncacace** [comment]
I agree, I would definitely add a tiny gif animation in my masthead (to match my website) if I could - staying conservative with the bandwidth of course.

<img width="74" alt="img2" src="https://github.com/user-attachments/assets/382f9a1c-8a12-4c99-bde4-a04d8f3a7675">
<img width="95" alt="img1" src="https://github.com/user-attachments/assets/138ef549-7374-434e-8785-67d9138b873d">
